Makefile

```diff
@@ -83,19 +83,19 @@ test-static: tidy test-merge test-dry test-no-wait_idle test-unused-modules test
 ifeq ($(TESTS),compile)
 test: test-compile
 else
-test: unit-test test-static test-compile
+test: unit-test test-static test-compile perlcritic
 endif
 
-PERLCRITIC=PERL5LIB=tools/lib/perlcritic:$$PERL5LIB perlcritic --quiet --gentle --include Perl::Critic::Policy::HashKeyQuote --include Perl::Critic::Policy::ConsistentQuoteLikeWords
+PERLCRITIC=PERL5LIB=tools/lib/perlcritic:$$PERL5LIB perlcritic --quiet --gentle
 
 .PHONY: perlcritic
 perlcritic: tools/lib/
-	${PERLCRITIC} .
+	${PERLCRITIC} ./tests ./products ./lib
```

Comment:
that I consider unsafe because whenever we add a new directory we probably forget to update this line. If this is about excluding the "os-autoinst" checkout please use the `--exclude` option. Or try `$(git ls-files '*.p[ml]')`. I tried your list and the "git ls-files" and the first took 51s and the second 50s so comparable.